sdks/python/container/boot.go

```diff
@@ -185,6 +185,7 @@ func main() {
 	os.Setenv("SEMI_PERSISTENT_DIRECTORY", *semiPersistDir)
 	os.Setenv("LOGGING_API_SERVICE_DESCRIPTOR", proto.MarshalTextString(&pipepb.ApiServiceDescriptor{Url: *loggingEndpoint}))
 	os.Setenv("CONTROL_API_SERVICE_DESCRIPTOR", proto.MarshalTextString(&pipepb.ApiServiceDescriptor{Url: *controlEndpoint}))
+	os.Setenv("RUNNER_CAPABILITIES", strings.Join(" ", info.GetRunnerCapabilities()))
```

Comment:
Same here. Parameter order is wrong for strings.Join. list then sep.

Though, until the Go proto update is patched in, it still won't compile.

Reply:
Thanks. Also patched in your proto generation commit. PTAL.